site/app/libraries/TokenManager.php

```diff
@@ -27,6 +27,9 @@ class TokenManager {
     private static $issuer;
 
     public static function initialize(string $secret, string $issuer): void {
+        if (mb_strlen($secret) < 64) {
+            throw new \LengthException('Invalid secret length, expect at least 64 characters, got ' . mb_strlen($secret) . ' characters');
+        }
```

Comment:
This is the only real impactful change in this PR, where `lcobucci/jwt` in 4.1.0 added some stricter validation so that it would prevent using a secret that wasn't compatible with the signer algorithm, in this case requiring that we use at least 64-bits. While we didn't do this for the test, we've always had the configuration of the secrets.session configuration setting as 64-bits (see #3668), unless it was manually changed to have less bits.

@bmcutler It would be useful to confirm that the value for `session` key in the `secrets.json` config file is at least 64 characters long.

Reply:
Confirmed that RPI has not changed the secret manually to have less than 64 characters.